packages/cojson/src/tests/coList.test.ts

```diff
@@ -128,11 +126,11 @@ test("appendItems at index", () => {
   content.appendItems(["hello", "world"], 0, "trusting");
   expect(content.toJSON()).toEqual([
     "first",
-    "hello",
-    "world",
     "second",
     "third",
     "fourth",
+    "hello",
+    "world",
   ]);
```

Comment:
This test is most likely what induced the reverse successors thing.

So it looks like that by fixing the conflicts management we can break the `splice` operation.